SharedObjects/Cards.cs

```diff
@@ -6,14 +6,14 @@
 
 namespace SharedObjects
 {
-    public class Cards
+    public class Cards //not sure what this class is for
```

Comment:
Yeah, I did a really sloppy outline. If anything doesn't seem to fit you can delete and change as needed. You can probably delete both the card and Cards class next push, now that we have the enum instead